src/olympia/lib/settings_base.py

```diff
@@ -1519,6 +1524,15 @@ def lazy_langs(languages):
 # Read-only mode setup.
 READ_ONLY = env.bool('READ_ONLY', default=False)
 
+# Expected retry-time that can be respected by clients. This will
+# be set as `Retry-After` header.
+# Please set this to a `datetime.timedelta()` instance that is
+# reasonable for clients to try again.
+# We don't support hard dates as these are usually quite hard to
+# adhere anyway.
+# Will be ignored if `None`
+READ_ONLY_RETRY_AFTER = None
```

Comment:
How is this supposed to be set? (by ops?)

I feel like we need some documentation somewhere.

Reply:
Yeah, this is supposed to be set by ops, similarly to how they set the read-only state. I'm not sure if we have general documentation about the read-only state, I couldn't find any so far.

I added a bit more stuff in f15ff7eaee, or do you mean we need some real docs in `docs/` at some point? Happy to do that in a follow-up commit.

Reply:
Yeah, I meant actual docs - like in our addons RTD docs, or mana.  For AMO, it's a pretty non-standard way of turning a setting on (i.e. normally ops add the setting to the env file and deploy; or outside of deploys we use waffle, or a management command).

Reply:
Yeah, the read-only state always was very different because it modifies the `DATABASES` setting. There may be other ways to do this, but they're out of scope for this issue imho.